editor/editor_properties.cpp

```diff
@@ -1198,6 +1275,11 @@ EditorPropertyFloat::EditorPropertyFloat() {
 
 ///////////////////// EASING /////////////////////////
 
+void EditorPropertyEasing::set_read_only(bool p_read_only) {
+	spin->set_read_only(p_read_only);
+	EditorProperty::set_read_only(p_read_only);
+};
```

Comment:
This doesn't seem to have any effect, the `spin` is still editable after it appears. Might be some bug.

But there is another thing. The curve box is still editable. You should probably ignore the input if the property is readonly. This will also fix the spin slider problem, because it will never appear.